src/Bravo3/OrmBundle/Services/Factories/OrmFactory.php

```diff
@@ -15,11 +15,12 @@ class OrmFactory
      * @param string          $cache_dir
      * @return EntityManager
      */
-    public static function createEntityManager(DriverInterface $driver, $cache_dir)
+    public static function createEntityManager(DriverInterface $driver, $cache_dir, $hydration_exceptions_as_events)
```

Comment:
Doesn't this get a default value of false via this though? https://github.com/bravo3/orm-bundle/pull/2/files#diff-3b89a603bd5905789f4b5e2ede10db75R50. Still, it probably reads better if I also add false here.